lib/main.dart

```diff
@@ -483,7 +482,7 @@ class _FritterAppState extends State<FritterApp> {
           blendOnColors: false,
         ),
         visualDensity: FlexColorScheme.comfortablePlatformDensity,
-        useMaterial3: false,
+        useMaterial3: true,
```

Comment:
I don't think I'm ready to turn this back on in Fritter, until Flutter's Material 3 support is fully baked. Last time I enabled this, it made some bizarre layout changes, and there were plenty of missing components.

See c25c02c

Reply:
Ok, I understand I will revert this change